internal/commands/config/config.go

```diff
@@ -0,0 +1,66 @@
+package config
+
+import (
+	"fmt"
+
+	"github.com/buildpacks/pack/internal/commands"
+
+	"github.com/spf13/cobra"
+
+	"github.com/buildpacks/pack/internal/config"
+	"github.com/buildpacks/pack/logging"
+)
+
+func Config(logger logging.Logger, cfg config.Config, cfgPath string) *cobra.Command {
```

Comment:
```suggestion
func NewConfigCommand(logger logging.Logger, cfg config.Config, cfgPath string) *cobra.Command {
```

Reply:
I'm happy to change that, but in my initial comment 
 to the stack cmd PR (https://github.com/buildpacks/pack/pull/937#discussion_r518652724), I had urged away from doing that. 

> Our command pattern is typically to do the command name itself (e.g. command.Build), so I think this would fit the pattern a bit more.

Just want to confirm whether this should be the new pattern. It does make more sense (because then it will be used as `rootCmd.AddCommand(config.NewCommand(...))`